python/cudf/cudf/core/column/lists.py

```diff
@@ -171,6 +171,12 @@ def deserialize(cls, header, frames):
             size=header["size"],
         )
 
+    @property
+    def __cuda_array_interface__(self):
+        raise NotImplementedError(
+            "Lists are not yet supported via `__cuda_array_interface__`"
+        )
```

Comment:
Do we need this for decimal columns as well?

Reply:
For decimal columns, it seems to be working but not sure how it works interoperably with other libraries:
```python
>>> import cudf
>>> s = cudf.Series([1, 2], dtype=cudf.Decimal64Dtype(10,2))
>>> s.__cuda_array_interface__
{'shape': (2,), 'strides': (8,), 'typestr': 'decimal64(10, 2)', 'data': (140550938296832, False), 'version': 1}
```

Reply:
No one will be able to handle the `typestr` since it's not standardized. I think we should similarly disable this.

Reply:
Done 👍